ide/editor.completion/src/org/netbeans/modules/editor/completion/PatchedHtmlRenderer.java

```diff
@@ -841,7 +841,7 @@ If NO (it's a closing tag)
                             //Word wrap mode
                             goToNextRow = true;
 
-                            int lastChar = new Double(nextTag - estCharsOver).intValue();
+                            int lastChar = Double.valueOf(nextTag - estCharsOver).intValue();
```

Comment:
int minus double, result (double) is cast to int
->
`(int) (nextTag - estCharsOver)`